<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Генератор Параметров</string>
  </property>
  <widget class="QPushButton" name="Confirmation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Поза</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Animal">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Животное</string>
   </property>
  </widget>
  <widget class="QLabel" name="Gender">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Пол</string>
   </property>
  </widget>
  <widget class="QLabel" name="Emotion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Эмоция</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddAnimal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Животное</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddEmotion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Эмоцию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Позу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
